pkg/mutation/assign_mutator.go

```diff
@@ -132,6 +132,10 @@ func (m *AssignMutator) DeepCopy() types.Mutator {
 	return res
 }
 
+func (m *AssignMutator) String() string {
+	return fmt.Sprintf("%s/%s/%s:%s", m.id.Kind, m.id.Namespace, m.id.Name, m.assign.GetResourceVersion())
```

Comment:
`m.assign.GetResourceVersion()` - would using Generation not be better? Since it's consecutive numbers it would be much easier to follow.